Detector/DetFCCeeECalInclined/compact/FCCee_ECalBarrel_thetamodulemerged.xml

```diff
@@ -0,0 +1,159 @@
+    <constant name="Steel_thickness" value="0.1*mm"/>
+    <!-- total amount of glue in one passive plate: it is divided for the outside layer on top and bottom -->
+    <constant name="Glue_thickness" value="0.1*mm"/>
+    <!-- readout in between two absorber plates -->
+    <constant name="readout_thickness" value="1.2*mm"/>
+  </define>
+
+  <display>
+    <vis name="ecal_envelope" r="0.1" g="0.2" b="0.6" alpha="1" showDaughers="false" visible="true" />
+  </display>
+
+  <readouts>
+    <!-- readout for the simulation -->
+    <!-- offset in theta is the eta max value including cryostat TO BE FIXED -->
+    <!-- THIS LACKS Z POSITION -->
+    <!--readout name="ECalBarrelTheta">
+        <segmentation type="GridTheta" grid_size_theta="0.009817477/4" offset_theta="0.5902785"/>
+        <id>system:4,cryo:1,type:3,subtype:3,layer:8,module:11,theta:10</id>
+    </readout-->
+
+    <!-- readout for the reconstruction, with phi-theta -->
+    <!-- readout name="ECalBarrelPhiTheta">
+        <segmentation type="FCCSWGridPhiTheta" grid_size_theta="0.009817477/4" phi_bins="1545" offset_theta="0.58902785" offset_phi="-pi+(pi/1545.)"/>
+        <id>system:4,cryo:1,type:3,subtype:3,layer:8,phi:11,theta:10</id>
+    </readout -->
+
+    <!-- readout for the simulation, with no merged modules and no merged cells in theta -->
+    <readout name="ECalBarrelModuleThetaMerged">
+      <!--segmentation type="FCCSWGridModuleThetaMerged" nModules="1545" mergedCells_Theta="1 1 1 1 1 1 1 1 1 1 1 1" mergedModules="1 1 1 1 1 1 1 1 1 1 1 1" grid_size_theta="0.00981745/4" offset_theta="0.5902785"/-->
+      <segmentation type="FCCSWGridModuleThetaMerged" nModules="1545" mergedCells_Theta="4 1 4 4 4 4 4 4 4 4 4 4" mergedModules="2 2 2 2 2 2 2 2 2 2 2 2" grid_size_theta="0.009817477/4" offset_theta="0.5902785"/>
+        <id>system:4,cryo:1,type:3,subtype:3,layer:8,module:11,theta:10</id>
+    </readout>
+
+    <!-- readout for the reconstruction, with merged modules and merged cells in theta -->
+    <readout name="ECalBarrelModuleThetaMerged2">
+      <segmentation type="FCCSWGridModuleThetaMerged" nModules="1545" mergedCells_Theta="2 4 2 1 2 1 2 2 1 1 1 2" mergedModules="2 1 1 2 2 1 1 1 2 2 1 1" grid_size_theta="0.009817477/4" offset_theta="0.5902785"/>
+        <!--segmentation type="FCCSWGridModuleThetaMerged" nModules="1545" mergedCells_Theta="4 1 4 4 4 4 4 4 4 4 4 4" mergedModules="2 2 2 2 2 2 2 2 2 2 2 2" grid_size_theta="0.009817477/4" offset_theta="0.5902785"/-->
+        <id>system:4,cryo:1,type:3,subtype:3,layer:8,module:11,theta:10</id>
+    </readout>
+  </readouts>
+
+  <detectors>
+    <!--detector id="BarECal_id" name="ECalBarrel" type="EmCaloBarrelInclined" readout="ECalBarrelEta"-->
+    <!-- detector id="BarECal_id" name="ECalBarrel" type="EmCaloBarrelInclined" readout="ECalBarrelTheta"-->
+    <detector id="BarECal_id" name="ECalBarrel" type="EmCaloBarrelInclined" readout="ECalBarrelModuleThetaMerged">
```

Comment:
Use the newly created detector type (see comment on the detector builder name) to avoid breaking previous configurations

Reply:
see answer to previous comment.